New package: ProteinChains v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: ProteinChains
- Repository: https://github.com/MurrellGroup/ProteinChains.jl
- Created by: @AntonOresten
- Version: v0.0.1
- Commit: 67078b836ba6db7fdb8023af4d43a5801263d5ed
- Reviewed by: @AntonOresten
- Reference: https://github.com/MurrellGroup/ProteinChains.jl/commit/67078b836ba6db7fdb8023af4d43a5801263d5ed#commitcomment-146175638
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Add `ProteinChain` type
- Add `ProteinStructure` type
- Add `BackboneGeometry` type
- Add `IdealResidue` type
- Deploy documentation
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd66ae3267d23c33f23d9187511f86d4bc0829262aa381426f2a6a564f7ed6d30b6b797170bd9b76b8d3643813fb14b2a9692bd1742f1e0752eec96ee3865ae97869fc139f33ff193741e5fb017ac0b522eae15f323dcffc6b9693fe50e0f996019142e144e505ee171e0fdda8ae3e559b4af28824ac901ce4808648ec3a36eda1af89c6dd10f0652fea5aa09bc00dcce515d15d54ba23868e1f4c2bbcdbc6287ba32b990e95b50ab5e3528cddcb1fa0409 -->

goerz commented:
Thank you for submitting your package! It seems like the documentation links in the README currently do not work. The material currently in the README itself isn’t quite enough. Can you either fix the links or add some more documentation (like a usage example) to the README?

JuliaRegistrator committed:
New package: ProteinChains v0.0.1

UUID: b8e8f2a5-48d3-44f1-ba0d-c71cb7726ff8
Repo: https://github.com/MurrellGroup/ProteinChains.jl.git
Tree: c5cc6d301eb55964f7664105441c8d77c4ad8b5c

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
